### DISCUSSION TITLE
Using GE for setting expectations on the nested JSON file

### DISCUSSION THREAD

**Discussion by jagsanand** [great-expectations/great_expectations#8516] [Support]
Hi All,
Need help regarding setting expectations on the nested JSON.
My JSON is on following type:
data = {
    "id": "1234",
    "version": 0,
    "case": "123",
    "category": "abcd",
    "pat_id": "1234",
    "notes": {
        "version": "V1.0",
        "questions": [
            {
                "q": "bmi",
                "snomed": "",
                "loinc": "",
                "value": "123"
            },
            {
                "q": "cancer_stage",
                "snomed": "",
                "loinc": "",
                "value": "1"
            }
        ]
    }
}

I want to set an expectation on "bmi" values to be between 20-30 for example.
I could achieve top level expectation setting on column ( in this case "pat_id" attribute but not for nested structure"

Any help is appreciated. thanks.

**Comment by shivamka1** [comment]
https://github.com/great-expectations/great_expectations/issues/3975#issuecomment-1458682543
